docs/docs/dev_docs/tutorials/token_portal/depositing_to_aztec.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Depositing Tokens to Aztec
+---
+
+In this step, we will write our token portal contract on L1.
+
+## Initialize Solidity contract
+
+In `l1-contracts/contracts` create a new file called `TokenPortal.sol` and paste this:
+
+```solidity
+import {IERC20} from "@oz/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@oz/token/ERC20/utils/SafeERC20.sol";
+
+// Messaging
+
+import {IRegistry} from "@aztec/core/interfaces/messagebridge/IRegistry.sol";
```

Comment:
none of this will work. There is no `@aztec` package to import from...
It will work after https://github.com/AztecProtocol/aztec-packages/issues/2453 though

Reply:
fixed all these